doc/edoc/gendoc.erl.in

```diff
@@ -36,7 +36,9 @@ main([LibraryName, SrcDir, TgtDir]) ->
             {doclet, edown_doclet},
             {source_path, ["src"]},
             {library, LibraryName},
-            {LibraryName, "https://atomvm.net/doc/apidocs/erlang/" ++ LibraryName},
+            % This is used as Canonical Link Relation
+            % See also: https://datatracker.ietf.org/doc/html/rfc6596.html
+            {LibraryName, "https://doc.atomvm.org/release-0.6/apidocs/erlang/" ++ LibraryName},
```

Comment:
It seems like we would want to use main rather than release-0.6 so we don’t loose SEO during new release cycles.

Reply:
I see, so this will not be forwarded and main will use main for the html_baseurl.